Targets/SetTargetABCACD.php

```
@@ -0,0 +1,375 @@
+    $numClubs    = count($clubList);
+
+    if ($numClubs === 0) return [$assignments, $unassigned];
+
+    // Club 0 (largest) → A column: always wave 1 on every boss
+    $idxA = 0;
+    foreach ($clubList[0] as $athlete) {
+        if ($idxA < count($colA)) {
+            $assignments[$colA[$idxA++]] = $athlete;
+        } else {
+            $unassigned[] = $athlete;
+        }
```

PHPMD (2.15.0)

[error] The method pl_abc_acd_assign uses an else expression. Else clauses are basically not necessary and you can simplify the code by not using them. (undefined)

(ElseExpression)